### DISCUSSION TITLE
searx as default engine on Chrome (Vivaldi)

### DISCUSSION THREAD

**Discussion by gerardvignali** [searx/searx#2911] [General]
On Chrome (Vivaldi) with searx as default engine, a query in the address or query field opens the searx page but with an empty search field

**Comment by cyclaero** [comment]
Searxing from the address bar does work here with Vivaldi on Windows and Mac. Please verify the URL which you specified in Vivaldi’s search settings.

Here I got: `http://searx.mylocal.domain?q=%s`
In addition I enabled the option to use POST requests for searching.
